plugins/com.jboss.devstudio.core.cheatsheet/src/com/jboss/devstudio/core/cheatsheet/CheatsheetStartup.java

```diff
@@ -0,0 +1,14 @@
+package com.jboss.devstudio.core.cheatsheet;
+
+import org.eclipse.ui.cheatsheets.OpenCheatSheetFromHelpAction;
+
+public class CheatsheetStartup implements org.eclipse.ui.IStartup{
+
+	@Override
+	public void earlyStartup() {
+		OpenCheatSheetFromHelpAction action = new OpenCheatSheetFromHelpAction();
```

Comment:
Not sure to understand. What will be the side-effect to normal users of DevStudio ? Will they be impacted ?

Reply:
I would guess that OOTB all devstudio users will, in addition to Central opening automatically, will also get the Cheatsheet view opened.